cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -226,7 +226,8 @@ impl pallet_balances::Config for Runtime {
 	type FreezeIdentifier = ();
 	// We allow each account to have holds on it from:
 	//   - `NftFractionalization`: 1
-	type MaxHolds = ConstU32<1>;
+	//   - `StateTrieMigration`: 1
+	type MaxHolds = ConstU32<2>;
 	type MaxFreezes = ConstU32<0>;
```

Comment:
Does anyone know the reason for manually maintaining this number? Why not just set it to something like 1024?

Reply:
It increases PoV. We should be able to add an `integrity_test` for it though.

Reply:
there is an `integrity_test` for checking for correct minimum:
https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/balances/src/lib.rs#L552-L557

but I think it checks only `VARIANT_COUNT` and not the `sum_of(VARIANT_COUNT of all HoldReason)`, I am writing a test for that to be sure

Reply:
I've summarized problem with that `integrity_test` check here: https://github.com/paritytech/polkadot-sdk/pull/2657

Reply:
I did little investigation https://github.com/paritytech/polkadot-sdk/pull/2657 and `MaxHolds` is removed there, so nobody need to guess and care about setting correct value for `MaxHolds` anymore